eng/versioning/version_client.txt

```diff
@@ -55,15 +55,15 @@ com.azure:azure-spring-boot-test-core;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-keyvault;1.0.0;1.0.0
 com.azure:azure-spring-data-gremlin;2.3.1-beta.1;2.3.1-beta.1
-com.microsoft.azure:azure-active-directory-b2c-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-active-directory-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-cosmosdb-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-data-gremlin-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-spring-boot-metrics-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
-com.microsoft.azure:azure-spring-boot;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-active-directory-b2c-spring-boot-starter;2.3.3-beta.1;2.3.3
```

Comment:
Previously, the dependency and current values were the same because these are effectively new libraries (you've got different parentage, different build rules etc.) in our repository. Since this is the first release out of the repository, there is no dependency version that matches what these dependency versions are set to, they should both be the same value until after this release.

@jialindai @saragluna - As I'd mentioned above, being that these are effectively new libraries in our repository with new parentage and build rules etc., should this just be a hotfix update or at least a minor update?